avaframe/out3Plot/statsPlots.py

```diff
@@ -298,7 +300,12 @@ def plotProbMap(avaDir, inDir, cfgFull, demPlot=False):
 
         # create figure and add title
         fig = plt.figure(figsize=(pU.figW*2, pU.figH))
-        fullTitle = cfg['name'] + ' based on %s $>$ %s %s' % (cfgFull['GENERAL']['peakVar'], cfgFull['GENERAL']['peakLim'], cfgFull['GENERAL']['unit'])
+
+        fullTitle = '%s %s based on %s $>$ %s %s' % (avaName,
```

Comment:
or shall we add the avaName on the plot (inside) instead of making the title longer?

Reply:
Mh, for the cases I had it worked out fine, but we can also move it to the inside of the plot.